examples/run_example.py

```diff
@@ -13,3 +13,4 @@
 total_time = (end_time - start_time).total_seconds()
 
 print(f"Total time taken: {total_time}, result: {result.outputs_formatted()}")
+print(f"Error: {result.error}")
```

Comment:
I personally use this for printing errors:
```suggestion
if result.error:
    print(f"Error: {result.error.json()}")
else:
    print(
        f"Total time taken: {total_time}\n"
        f"Run ID: {result.id}\n"
        f"Result: {result.outputs_formatted()}"
    )
```